src/nvim/tui/tui.c

```diff
@@ -459,8 +459,8 @@ static void update_attrs(UI *ui, HlAttrs attrs)
   bool underline = attr & (HL_UNDERLINE), undercurl = attr & (HL_UNDERCURL);
 
   if (unibi_get_str(data->ut, unibi_set_attributes)) {
```

Comment:
For me this condition appears to be true always, which results in the `else` below not being reached (where standout is set).